docs/root/operations/cli.rst

```diff
@@ -287,6 +287,11 @@ following are the command line options that Envoy supports.
   and these occurrences are counted in the :ref:`server.dynamic_unknown_fields <server_statistics>`
   statistic.
 
+.. option:: --ignore-unknown-dynamic-fields
+
+  *(optional)* This flag disables validation of protobuf configuration
+   for unknown fields in dynamic configuration, and ignores GENERA write more.
```

Comment:
Nup, we still have some of these GENERA write mores! :)

Reply:
I think I'll choose to be glad past-pgenera was so thorough rather than be dismayed present-pgenera is situationally blind. :)